### PR TITLE
feat(Kubernetes): Add helm install instructions

### PR DESCRIPTION
**Pull request by paperclypse**
Added in response to infosec-10264

### CODE REVIEW

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 1]
```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
 
 <InstallFeedback />
 
+## Install Fluent Bit output plugin [#fluent-bit]
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:42]: @paperclypse how is this different from our separate plug-in doc about Fluent Bit: https://docs.newrelic.com/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding/

Why is Fluent Bit being singled out here in this context? Why aren't we mentioning other log forwarders here? 

Previously, this Kubernetes plug-in doc was fine without a reference to the Fluent Bit plug-in. What changed?

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 2]
```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
...
+
+New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:44]: @paperclypse it's my understanding that a DaemonSet just makes sure a pod runs on certain nodes. The log forwarder sounds like a plug-in. Could you confirm with our devs on this?

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 3]
```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
...
+
+You can install it in your cluster using our Helm chart.
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:46]: @paperclypse I'd use some h3s to clarify that you're offering two Helm approaches. To set that up, I'd suggest a sentence like this: "You can install it in your cluster using our Helm chart in two ways."

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 4]
```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
...
+
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:47]: @paperclypse you could add an h3: `### Use our guided install [#helm-guided-install]

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 5]
```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
...
+Despite the [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart being able to work standalone, we recommend installing it as part of the [nri-bundle](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart.
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:48]: This sentence was tough for me to follow. Consider a revision like this: Although the [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart works as a standalone, we recommend installing it as part of the [nri-bundle](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. The best way to do this is through...."

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 6]
```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
...
+## Install Fluent Bit output plugin [#fluent-bit]
+
+New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
+
+You can install it in your cluster using our Helm chart.
+
+Despite the [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart being able to work standalone, we recommend installing it as part of the [nri-bundle](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. 
+
+The best way to install this is through our [guided installation](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/) process. This guided install can generate the Helm 3 commands required to install it ([see "Helm 3"](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#finish-your-install)). 
+
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:51]: You could add an h3: `###Manual installation [#manual-helm-install]

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx [hunk 7]
```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
...
+Alternately, you can [install it manually using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#install-k8-helm).
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:52]: @paperclypse the link in this sentence is for the guided installation, but in this section we only want to give manual instructions, right?

**Review by rhetoric101 — COMMENTED** [review]
@paperclypse I inserted a few suggestions. See what you think.

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:44]: This content came from our dev's readme, so I assume that it's accurate.

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx (hunk 5 above)
Comment (paperclypse) [reply src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:48]: much better, thanks!

File: src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx (hunk 7 above)
Comment (paperclypse) [reply src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx:52]: Actually, I think I'm going to remove that link there. I think it distracts from the CLI commands there.

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**